internal/worker/sshserver/server.go

```diff
@@ -426,6 +427,9 @@ func (s *ServerWorker) newTerminatingSSHServer(ctx ssh.Context, destination virt
 		Handler: func(session ssh.Session) {
 			proxier.SessionHandler(session)
 		},
+		SubsystemHandlers: map[string]ssh.SubsystemHandler{
```

Comment:
How does the flow work here? Reverse connection established with a session, then inside this particular handler we create the session, then open the sftp subsystem?

Reply:
Yip, just as you've described. A subsystem is just an invocation of a specific program see [here](https://superuser.com/questions/425396/whats-the-principle-behind-openssh-subsystems-or-command-option). So it's the same flow as the session handler essential.